commit be2b34fa9be31c60a95989f984c9a5d40cd781b6
Author: Kees Cook <kees@kernel.org>
Date:   Tue May 3 13:55:02 2022 -0700

    randstruct: Move seed generation into scripts/basic/
    
    To enable Clang randstruct support, move the structure layout
    randomization seed generation out of scripts/gcc-plugins/ into
    scripts/basic/ so it happens early enough that it can be used by either
    compiler implementation. The gcc-plugin still builds its own header file,
    but now does so from the common "randstruct.seed" file.
    
    Cc: linux-hardening@vger.kernel.org
    Signed-off-by: Kees Cook <keescook@chromium.org>
    Link: https://lore.kernel.org/r/20220503205503.3054173-6-keescook@chromium.org

diff --git a/scripts/gcc-plugins/gen-random-seed.sh b/scripts/gcc-plugins/gen-random-seed.sh
deleted file mode 100755
index 68af5cc20a64..000000000000
--- a/scripts/gcc-plugins/gen-random-seed.sh
+++ /dev/null
@@ -1,9 +0,0 @@
-#!/bin/sh
-# SPDX-License-Identifier: GPL-2.0
-
-if [ ! -f "$1" ]; then
-	SEED=`od -A n -t x8 -N 32 /dev/urandom | tr -d ' \n'`
-	echo "const char *randstruct_seed = \"$SEED\";" > "$1"
-	HASH=`echo -n "$SEED" | sha256sum | cut -d" " -f1 | tr -d ' \n'`
-	echo "#define RANDSTRUCT_HASHED_SEED \"$HASH\"" > "$2"
-fi